.github/workflows/main.yml

```diff
@@ -0,0 +1,148 @@
+name: main
+
+on: [push, pull_request]
+
+jobs:
+  no-naked-pointers:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: configure tree
+      run: ./configure --disable-naked-pointers --disable-stdlib-manpages
+    - name: Build
+      run: |
+        make -j world.opt
+    - name: Run the testsuite
+      run: |
+        make -C testsuite USE_RUNTIME=d all
+  flambda:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: configure tree
+      run: |
+        XARCH=x64 CONFIG_ARG=--enable-flambda OCAMLRUNPARAM=b,v=0 bash -xe tools/ci/actions/runner.sh configure
+    - name: Build
+      run: |
+        XARCH=x64 CONFIG_ARG=--enable-flambda OCAMLRUNPARAM=b,v=0 bash -xe tools/ci/actions/runner.sh build
+    - name: Run the testsuite
+      run: |
+        XARCH=x64 CONFIG_ARG=--enable-flambda OCAMLRUNPARAM=b,v=0 bash -xe tools/ci/actions/runner.sh test
+    - name: Install
+      run: |
+        XARCH=x64 CONFIG_ARG=--enable-flambda OCAMLRUNPARAM=b,v=0 bash -xe tools/ci/actions/runner.sh install
+    - name: Other checks
+      run: |
+        XARCH=x64 CONFIG_ARG=--enable-flambda OCAMLRUNPARAM=b,v=0 bash -xe tools/ci/actions/runner.sh dither
```

Comment:
I don't know what `dither` means. Could we use a simpler name for this action? (For example `other-checks`?)

Reply:
Oops - that should have gone because I first pushed!